internal/cmd/config/config.go

```diff
@@ -498,6 +527,36 @@ func Parse(d string) (*Config, error) {
 			}
 		}
 
+		workerStatusGracePeriod := result.Controller.WorkerStatusGracePeriod
+		if workerStatusGracePeriod == nil {
```

Comment:
should this be a call to `util.IsNil(...)` since it's an interface?

Reply:
This comment sort of applies to all these new fields which are empty interfaces.

Reply:
Done